maths/allocation_number.py

```diff
@@ -0,0 +1,52 @@
+#-*- coding:utf-8 -*-
+
+
+def allocation_num(num, x) -> int:
```

Comment:
Please see the __mypy__ output in the Travis CI log
https://github.com/TheAlgorithms/Python/blob/master/Travis_CI_tests_are_failing.md

Reply:
Line 4 says that the function returns an `int` but line 26 says that we expect a `list` of `str`.  Line 26 is correct and line 4 is wrong.

$ __mypy --ignore-missing-imports .__
```
maths/allocation_number.py:51: error: Incompatible return value type (got "List[str]", expected "int")
Found 1 error in 1 file (checked 95 source files)
The command "mypy --ignore-missing-imports ." exited with 1.
```